docs/rest-api.md

```diff
@@ -275,6 +276,10 @@ trades
 
         :param limit: Limits trades to the X last trades. No limit to get all the trades.
 
+trade
+	Return specific trade.
+        :param tradeid: Specify which trade to get.
+
 version
```

Comment:
This section is the output of `scripts/rest_client.py --show`.

I don't mind having the descriptions reworded / improved - but it should really happen in that script and then copy/pasting the output to here- not just in the docs ;) 
same applies to the new endpoint - which is also a "doc only" point - which is not really available in the script (at the moment ...).